### ISSUE TITLE
Server asked us to submit HIP report with md5sum

### ISSUE THREAD

**Issue by alex-k86** [yuezk/GlobalProtect-openconnect#56]
Hi! I am using saml authentication but after connection I can not access any resource in vpn. In the log I see the warning about md5sum of HIP report.
```
2021-04-29 09:31:44.174 INFO  [36636] [GPClient::onVPNLogAvailable@440] No MTU received. Calculated 1422 for ESP tunnel

2021-04-29 09:31:44.175 INFO  [36636] [GPClient::onVPNLogAvailable@440] POST https://some.server.com/ssl-vpn/hipreportcheck.esp

2021-04-29 09:31:44.372 INFO  [36636] [GPClient::onVPNLogAvailable@440] WARNING: Server asked us to submit HIP report with md5sum 1c9c4f1f793378a88e5be9711d3c8d21.
    VPN connectivity may be disabled or limited without HIP report submission.
    You need to provide a --csd-wrapper argument with the HIP report submission script.

2021-04-29 09:31:44.372 INFO  [36636] [GPClient::onVPNLogAvailable@440] Connected as xx.x.xxx.106, using SSL, with ESP in progress

2021-04-29 09:31:49.462 INFO  [36636] [GPClient::onVPNLogAvailable@440] Failed to connect ESP tunnel; using HTTPS instead.

```

**Comment by murph** [comment]
Hey @alex-k86,

I think there's two parts to your problem. First, you need some kind of HIP report script ([see the openconnect docs](https://www.infradead.org/openconnect/hip.html)). Second, you need the ability to set --csd-wrapper in this UI to call your script.

I just filed #70 for the second part. For the first, you'll probably need to figure that out / it's outside the scope of this project.

**Comment by yuezk** [comment]
The second part has been resolved in 1.3.0, as described in https://github.com/yuezk/GlobalProtect-openconnect#passing-the-custom-parameters-to-openconnect-cli
